libpod/healthcheck_linux.go

```diff
@@ -73,6 +73,14 @@ func (c *Container) removeTransientFiles(ctx context.Context) error {
 	defer conn.Close()
 	timerFile := fmt.Sprintf("%s.timer", c.ID())
 	serviceFile := fmt.Sprintf("%s.service", c.ID())
+
+	// If the service has failed (the healthcheck has failed), then
+	// the .service file is not removed on stopping the unit file. If
+	// we check the properties of the service, it will automatically
+	// reset the state.  But checking the state takes msecs vs usecs to
+	// blindly call reset and eat any errors
+	_ = conn.ResetFailedUnitContext(ctx, serviceFile)
```

Comment:
Should we at least logrus.Debug the errors?

Reply:
I think it could be misleading, but ill do as reviewers want ...

Reply:
If it returns expected errors like ENOEXST, then don't log those.  I am worried about weird errors like EPERM, which we might not notice.